### DISCUSSION TITLE
Is there a way to prerender?

### DISCUSSION THREAD

**Discussion by mayank99** [codesandbox/sandpack#457] [Q&A] [answered]
I was trying to dig in the docs for some guidelines/examples on prerendering the sandbox on server before the JS loads ("SSR"), but I couldn't find anything.

If I understand correctly, it should be possible to prerender the container and the code, and show a blank box instead of the iframe. Is there an example somewhere of doing this? It would be really useful to prevent layout shifts.

**Comment by danilowoz** [comment, marked as answer]
Hey @mayank99. It will be possible to SSR the editor component from the v1.x, which is currently in development here https://github.com/codesandbox/sandpack/pull/375 

I'll let you know as soon as we have a beta version of it.
